content/blog/2021-03-31-march-21-community-gems.md

```diff
@@ -0,0 +1,102 @@
+---
+title: March '21 Community Gems
+date: 2021-03-31
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-03-31/gems-cover.png
```

Comment:
💅 not this time, but let's try to find someone who can do these illustration for us